setup-di-compile/README.md

```diff
@@ -0,0 +1,29 @@
+# Magento 2 setup:di:compile action
+A Github Action that runs `php bin/magento setup:di:compile` and checks for compilation errors.
+
+## Inputs
+
+See the [action.yml](./action.yml)
+
+## Usage
+
+```yml
+name: Magento setup:di:compile check
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  setup-di-compile:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: mage-os/github-actions/setup-di-compile@main
+      with:
+        php_version: "8.2"
```

Comment:
```suggestion
        php_version: "8.3"
```

@rhoerr since mage-OS 1.0.6 runs on 8.3, should we use 8.3 here?

Reply:
@fballiano  done